docs/getting-started-guides/minikube.md

```diff
@@ -0,0 +1,76 @@
+---
+---
+
+* TOC
+{:toc}
+
+### Requirements
+
+Minikube requires that VT-x/AMD-v virtualization is enabled in BIOS.
```

Comment:
Link to Details / 1 line command  to determine if action is required?  Does this apply to Macs, or is this Linux only?